can/interfaces/pcan/pcan.py

```diff
@@ -62,26 +63,17 @@
 }
 
 
-pcan_fd_parameter_list = [
-    "nom_brp",
-    "nom_tseg1",
-    "nom_tseg2",
-    "nom_sjw",
-    "data_brp",
-    "data_tseg1",
-    "data_tseg2",
-    "data_sjw",
-]
-
-
 class PcanBus(BusABC):
     def __init__(
         self,
         channel="PCAN_USBBUS1",
         state=BusState.ACTIVE,
         bitrate=500000,
+        timing=None,
+        data_timing=None,
+        fd=False,
         *args,
-        **kwargs
+        **kwargs,
```

Comment:
Is there a reason for this extra comma?

Reply:
> You might have noticed that closing brackets are always dedented and that a trailing comma is always added. Such formatting produces smaller diffs; when you add or remove an element, it’s always just one line. Also, having the closing bracket dedented provides a clear delimiter between two distinct sections of the code that otherwise share the same indentation level (like the arguments list and the docstring in the example above).

See [The Black code style](https://black.readthedocs.io/en/stable/the_black_code_style.html#how-black-wraps-lines) (our formatter).

Reply:
From the link you sent:
"One exception to adding trailing commas is function signatures containing *, *args, or **kwargs. In this case a trailing comma is only safe to use on Python 3.6. Black will detect if your file is already 3.6+ only and use trailing commas in this situation. If you wonder how it knows, it looks for f-strings and existing use of trailing commas in function signatures that have stars in them. In other words, if you’d like a trailing comma in this situation and Black didn’t recognize it was safe to do so, put it there manually and Black will keep it."

Are we already 3.6+?